src/components/services/Services.js

```diff
@@ -41,6 +40,7 @@ type Props = {
  */
 export function Services(props: Props) {
   const { context } = props
+  const [account, setAccount] = useState<EdgeAccount | void>()
```

Comment:
Also, when the account changes, do we want to re-render the entire parent or just this line:
`{account == null ? null : <AccountCallbackManager account={account} />}`
If we want to re-render the parent then it's fine, but it we only want to render the child then maybe use 'useRef' for the account.

Reply:
We want to cause a re-render when the account changes, as this is the only way to send props to our children. Even if only a single child changes, you have to render the whole parent. The `useRef` hook never causes any rendering, so it's useless for anything other than effects.

Why doesn't the `AccountCallbackManager` use `useSelector` to grab the account? Because the account may not exist, and we can't put conditional logic around hooks.

We could put a wrapper around `AccountCallbackManager` that is just a single `useSelector` call, followed by a conditional render of the `AccountCallbackManager`. This seems overkill - the `account` only changes at login / logout time, so it would be a lot of work for a small optimization. However, I will do it if you ask.

Reply:
If you are certain there are no other "edge" cases where the account can change, then no need to change.